service/src/test/java/greencity/ModelUtils.java

```diff
@@ -3049,10 +3047,8 @@ public static Order getOrderForGetOrderStatusData2Test() {
                 .tariffLocations(Set.of(TariffLocation.builder()
                     .id(1L)
                     .build()))
-                .maxAmountOfBigBags(2L)
-                .maxPriceOfOrder(500000L)
-                .minAmountOfBigBags(99L)
-                .minPriceOfOrder(500L)
+                .maxQuantity(2L)
+                .minQuantity(99L)
```

Comment:
is it a valid case if `minQuantity` is less than `maxQuantity`

Reply:
Yes, I checked that was the same values